testmod-common/src/main/java/me/shedaniel/architectury/test/networking/NetworkTestEventHandler.java

```diff
@@ -0,0 +1,4 @@
+package me.shedaniel.architectury.test.networking;
+
+public class NetworkTestEventHandler {
```

Comment:
What's this?

Reply:
A dummy class, not sure if you wanted the example code to even do anything